### PR TITLE
Feat/ping 3319 check related follow anon users

### PR DESCRIPTION
**Pull request by brm-stnd**
Check fitur2 yang terkait get follower apakah terdampak atau tidak

Following (/profiles/following) - updated

Create new chat and add participant (/users/populate) - no need

Blocked user - no need

Onboarding: who to follow (/who-to-follow/list) - updated

Tagging (small suggestion list) (/mention/users/${name}} - no need

Discovery (user Search) (/discovery/init/user, /discovery/user) - no need

Profile Screen (/profiles/get-my-profile) - updated

Other Profile Screen (/profiles/get-other-profile-by-username) - updated

### CODE REVIEW

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 75778404284e69b7a156b5b00423f0da36f83d63 and 9cd477b5427677baf509e00257206bc854e99e6f commits.
</details>

<details>
<summary>Files selected (4)</summary>

* controllers/profiles/followAnonUser.js (1)
* controllers/profiles/following.js (1)
* controllers/profiles/getMyProfile.js (1)
* controllers/profiles/getOtherProfileByUsername.js (1)
</details>






---

File: controllers/profiles/following.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ module.exports = async (req, res) => {
       "user"."profile_pic_public_id" AS "user.profile_pic_public_id", 
       "user"."status" AS "user.status" FROM "user_follow_user" AS "UserFollowUser" 
       LEFT OUTER JOIN 
-      "users" AS "user" ON "UserFollowUser"."user_id_followed" = "user"."user_id" 
-      WHERE "UserFollowUser"."user_id_follower" = :id AND "UserFollowUser"."user_id_followed" != :id;`,
+      "users" AS "user" ON "UserFollowUser"."user_id_followed" = "user"."user_id"
+      WHERE "UserFollowUser"."user_id_follower" = :id AND "UserFollowUser"."user_id_followed" != :id AND "user"."is_anonymous" = 'false';`,
```
Comment (fajarism) [inline controllers/profiles/following.js:37]: this `false` should not use string

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 75778404284e69b7a156b5b00423f0da36f83d63 and 9cd477b5427677baf509e00257206bc854e99e6f commits.
</details>

<details>
<summary>Files selected (4)</summary>

* controllers/profiles/followAnonUser.js (1)
* controllers/profiles/following.js (1)
* controllers/profiles/getMyProfile.js (1)
* controllers/profiles/getOtherProfileByUsername.js (1)
</details>





<details>
<summary>Files not summarized due to errors (4)</summary>

* controllers/profiles/getOtherProfileByUsername.js (nothing obtained from openai)
* controllers/profiles/following.js (nothing obtained from openai)
* controllers/profiles/getMyProfile.js (nothing obtained from openai)
* controllers/profiles/followAnonUser.js (nothing obtained from openai)

</details>


<details>
<summary>Files not reviewed due to errors (4)</summary>

* controllers/profiles/followAnonUser.js (no response)
* controllers/profiles/getOtherProfileByUsername.js (no response)
* controllers/profiles/getMyProfile.js (no response)
* controllers/profiles/following.js (no response)

</details>


<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by fajarism — CHANGES_REQUESTED** [review]

File: controllers/profiles/following.js (hunk 1 above)
Comment (brm-stnd) [reply controllers/profiles/following.js]: fixed

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 9cd477b5427677baf509e00257206bc854e99e6f and be1f93644de0a5996d2b49a7d0f36ff8d7b7af0f commits.
</details>

<details>
<summary>Files selected (1)</summary>

* controllers/profiles/following.js (1)
</details>





<details>
<summary>Files not summarized due to errors (1)</summary>

* controllers/profiles/following.js (nothing obtained from openai)

</details>


<details>
<summary>Files not reviewed due to errors (1)</summary>

* controllers/profiles/following.js (no response)

</details>


<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Pull request merged**